index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+
+  <body>
+    <header class="header">
+      <h1>Quiz App</h1>
+    </header>
+
+    <!-- QUESTION CARDS -->
+    <section class="card">
+      <div class="centered">
+        <h2 class="card__heading">What property flips the axes in flexbox?</h2>
+        <button class="card__bookmark">📕</button>
+        <button class="card__button">Show Answer</button>
+      </div>
+      <div class="card__tag">
+        <button class="tag">&num;html</button>
+
+        <button class="tag">&num;flexbox</button>
+
+        <button class="tag">&num;css</button>
+      </div>
+    </section>
+
+    <section class="card">
+      <div class="centered">
+        <h2 class="card__heading">Lorem Ipsum dolor sit?</h2>
+        <button class="card__bookmark">📕</button>
+        <button class="card__button">Show Answer</button>
+      </div>
+      <div class="card__tag">
+        <button class="tag">&num;html</button>
+
+        <button class="tag">&num;flexbox</button>
+
+        <button class="tag">&num;css</button>
+      </div>
+    </section>
+  </body>
+
+  <footer class="footer">
+    <button class="footer__button">🏠</button>
+    <button class="footer__button">📕</button>
+    <button class="footer__button">👤</button>
```

Comment:
Since this is a navigation, please consider the HTML `nav` element with `<a>` tags.